src/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -81,7 +95,7 @@ describe('pdf-metadata', function () {
         fakePDFViewerApplication.metadata.get = sinon.stub().returns('dcTitle');
 
         return pdfMetadata.getMetadata().then(function (actualMetadata) {
-          assert.deepEqual(expectedMetadata, actualMetadata);
+          assert.deepEqual(actualMetadata, expectedMetadata);
```

Comment:
The arguments were the wrong way round here, producing confusing error messages.